lesson01/index.js

```diff
@@ -0,0 +1,2 @@
+let target = document.getElementById("js-target");
+target.innerHTML = "<li>これです</li>";
```

Comment:
MUST
If it is OK to use const, it is better to use const.
```suggestion
const target = document.getElementById("js-target");
target.innerHTML = "<li>これです</li>";
```

Reply:
Thanks for pointing that out.

I fixed.
[a8f7eca](https://github.com/tom106gunma/JavaScript-assignment/pull/1/commits/a8f7ecad05c4c3f71cadaa4e0e695f60224ef7d6)

Reply:
@groovywave 

Sorry for the delay in commenting.
I'll keep this file since I have a new template and I've been practicing Git.

[https://discord.com/channels/726252977928667179/726310768215195698/1090615383196700702](https://discord.com/channels/726252977928667179/726310768215195698/1090615383196700702)